### PR TITLE
feat: Add more projects

### PR DESCRIPTION
**Pull request by msrose**
https://github.com/tophat/commit-utils
https://github.com/tophat/monodeploy
https://github.com/tophat/commit-watch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -134,6 +134,48 @@ <h3 class="project-title">Codewatch</h3>
...
+                    </p>
+                </div>
+
+                <div class="project-card">
+                    <div class="project-logo">
+                        <img src="static/img/commit-utils.svg">
+                    </div>
+                    <h3 class="project-title">Commit Utils</h3>
+                    <p class="project-description">
+                        💬 Top Hat's commitlint config, including a commitizen adapter and conventional changelog preset
```
Comment (sanchitgera) [inline index.html]: Can we remove the emoji to be consistent with other descriptions?

File: index.html [hunk 2]
```diff
@@ -134,6 +134,48 @@ <h3 class="project-title">Codewatch</h3>
...
+                    </p>
+                </div>
+
+                <div class="project-card">
+                    <div class="project-logo">
+                        <img src="static/img/commit-watch.png">
+                    </div>
+                    <h3 class="project-title">commit-watch</h3>
+                    <p class="project-description">
+                        👀 Ensure your commit messages are formatted based on your commitlint config
```
Comment (sanchitgera) [inline index.html]: and here

File: index.html [hunk 3]
```diff
@@ -134,6 +134,48 @@ <h3 class="project-title">Codewatch</h3>
...
+                    </p>
+                    <p class="project-links">
+                        <a class="pink link" href="https://tophat.github.io/commit-watch">Website</a>
```
Comment (sanchitgera) [inline index.html]: Should we hold off on adding links to the website until we fill it in?

Comment (msrose) [reply index.html]: I feel we should just leave it in there, in all likelihood we won't get much traffic here until we actively promote the projects, so we can just make sure the websites are updated before we do that for each project.

**Comment by sanchitgera** [comment]
CR/QA LGTM

**Pull request merged**